templates/bot/csharp/command-and-response/CardModel.cs.tpl

```diff
@@ -0,0 +1,13 @@
+﻿namespace {{ProjectName}}
+{
+    using Newtonsoft.Json;
```

Comment:
Any concern if moving to the built-in library `System.Text.Json`?

Reply:
Both `botbuilder-dotnet` and `AdaptiveCards.Templating` SDK use the `Newtonsoft.Json` to (de)serialize JSON.  
e.g. 
- The [`expand`](https://github.com/microsoft/AdaptiveCards/blob/main/source/dotnet/Library/AdaptiveCards.Templating/AdaptiveCardTemplate.cs#L92) API in `AdaptiveCards.Templating` to render the JSON;
- The whole data chains (adaptiveCard content -> atttachment -> activity) are serialized/deserialized by `Newtonsoft.Json`

So I'd prefer to use `Newtonsoft.Json` to keep consistent and avoid unexpected error for serialization.